lib/IRGen/GenMeta.cpp

```diff
@@ -470,6 +470,7 @@ namespace {
     void addExtendedContext() {
       auto string = IGM.getTypeRef(
           E->getSelfInterfaceType()->getCanonicalType(),
+          E->getGenericSignature(),
```

Comment:
I wasn't sure about this one, but it might matter with @Azoy's plan for extensions to have their own generic parameters.

Reply:
I haven't poked around IRGen enough yet, but could you explain why this might affect extensions with there own generic parameters?

Reply:
Mostly because I'm relatively unfamiliar with it too. :-) The point I was trying to make was that this is probably already in the right form today since the extension can't have introduced any new parameters, but I'm not sure if that'll be true if it can. I think it still is, though.

(The latest change to the runtime, though, #25984, almost certainly is not correct in that world.)